packages/admin/src/Pages/Page.php

```diff
@@ -73,14 +78,6 @@ public static function getUrl(array $parameters = [], bool $absolute = true): st
         return route(static::getRouteName(), $parameters, $absolute);
     }
 
-    protected function notify(string $status, string $message): void
```

Comment:
Removing this method is a breaking change.

Reply:
It extracted into a Trait.

Reply:
Ah, yeah. Had that files collapsed for some reason.